src/components/main-form.js

```diff
@@ -80,22 +81,44 @@ const AmortizationTable = styled(Table)`
     border-left-color: ${GRAY}};
   }
 `
+const emphasize = cx('card', css(`border-color: ${LIGHT_GREEN};`))
+
+const box = bestOption => css`
+  background-color: ${bestOption ? LIGHT_GREEN : GRAY};
+  font-size: 3rem;
+  color: white;
+`
 
 /**
  * @param {*} props
- *   @property {boolean} highlight
+ *   @property {boolean} bestOption
  *   @property {Object} option
  */
 function ScenarioCol (props) {
-  const highlightClass = cx('card', css(`border-color: ${LIGHT_GREEN};`))
   return (
-    <Col className={`p-4 ${props.highlight && highlightClass}`}>
+    <Col className={`p-4 ${props.bestOption && emphasize}`}>
       <h5>{props.option.term} year scenario:</h5>
       <ol className={css('padding-inline-start: 1rem;')}>{props.children}</ol>
     </Col>
   )
 }
 
+/**
+ * @param {*} props
+ *   @property {boolean} bestOption
+ *   @property {Object} option
+ */
+function BoxCol (props) {
+  return (
+    <Col className={`p-4 text-center ${props.bestOption && emphasize}`}>
+      <div className={classNames(box(props.bestOption), 'p-3')}>
```

Comment:
Yeah, I think having the same padding as the summary would look better

Reply:
not so sure if I follow. Should i be changing the padding then? or did I get it right?

Reply:
Yeah it may help alignment if this was also `p-4`

Reply:
ah ok let me give that a try

Reply:
done. The Summary doesn't actually have padding right? It's the `ScenarioCol` that you were referring to?

Reply:
Ah yep